site/docs/v1/tech/apis/identity-providers/facebook.adoc

```diff
@@ -18,6 +18,7 @@ The email address returned by the Facebook Graph API will be used to create or l
 
 When the `picture` field is not requested FusionAuth will also call Facebook's `/me/picture` API to load the user's profile image and store it as the `imageUrl` in FusionAuth. When the `picture` field is requested, the user's profile image will be returned by the `/me` API and a second request to the `/me/picture` endpoint will not be required.
 
+:idp_long_lived_token_type: long-lived
```

Comment:
what is a `long-lived`?

Reply:
I guess that this makes sense, but I wonder if we should not offer an explainer -- it might be self-evident enough to a seasoned reader.

Reply:
It's a special facebook thing, we do link to the doco about it now.